plugins/woocommerce/src/Internal/ComingSoon/ComingSoonAdminBarBadge.php

```diff
@@ -65,10 +65,10 @@ public function output_css() {
 				#wpadminbar .quicklinks #wp-admin-bar-woocommerce-site-visibility-badge a.ab-item {
 					background-color: #F6F7F7;
 					color: black;
-					margin-top:6px;
+					margin-top:7px;
 					padding: 0 6px;
-					height: 20px;
-					line-height: 20px;
+					height: 18px;
+					line-height: 17px;
```

Comment:
```suggestion
					line-height: 18px;
```

I see an 18px-tall badge, but the text is not vertically centered. On my screen, it looks like the line height should be 18px. Do you see the same? Or is it just me?  🤔 

`line-height: 17px`
<img width="971" alt="Screenshot 2024-08-21 at 10 22 34" src="https://github.com/user-attachments/assets/a45e605e-c051-4a4c-a9e9-b83dc3e62086">

`line-height: 18px`
<img width="1093" alt="Screenshot 2024-08-21 at 10 24 30" src="https://github.com/user-attachments/assets/3b278f21-ad71-4551-a540-bbc9380973e9">

Reply:
Thanks @chihsuan. This is tricky, I think the 18px example causes the text to be un-balanced within the badge. However the 17px example causes the text to be offset from other text in the admin bar.

I think we need a more reliable approach than line-height. I'll merge for the code freeze but keep the issue open to follow-up on this.

Reply:
I think it might be possible to do something like display:inline-block and vertical-align: baseline